### DISCUSSION TITLE
续费专业版，订单查询显示ip不在白名单，专业版时长没有增加

### DISCUSSION THREAD

**Discussion by realwatch** [getlantern/lantern-forum-cn#895] [Q&A]
![IMG_0282](https://github.com/user-attachments/assets/2f804fe0-a6ac-4093-bd39-8b6d1928257d)
377233805@qq.com 请帮忙增加专业版时长

**Comment by MaxMKMax** [comment]
@realwatch OK 没问题. 我已经帮您提交上去了.

**Reply by realwatch** [reply]
mike 4天了还是没有生效 你那边提交后多久才能生效？

**Reply by realwatch** [reply]
@Mike777Khoo

**Reply by MaxMKMax** [reply]
@realwatch 您好. 非常抱歉让您久等了. 负责人刚好生病了, 在等待回复, 他这几天在休息. 我会帮您再问问看的. 谢谢您的耐心等待!
